en/news/_posts/2019-04-01-ruby-2-6-3-released.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: news_post
+title: "Ruby 2.6.3 Released"
+author: "naruse"
+translator:
+date: 2019-04-01 00:00:00 +0000
+lang: en
+---
+
+Ruby 2.6.3 has been released.
+
+This release update Unicode version to 12.1, which add to New Japanese Era "〓〓".
+
+This release also includes some bug fixes.
+See details [commit logs](https://github.com/ruby/ruby/compare/v2_6_2...v2_6_3).
+
+## Download
+
+
+
+
+## Release Comment
+
+Many committers, developers, and users who provided bug reports helped
+us make this release.
```

Comment:
Not a big deal, should we merge this with above line? 🙋

Reply:
meged!